study.md

```diff
@@ -19,8 +19,13 @@ Ember](https://www.codeschool.com/courses/try-ember) course from CodeSchool.
 
 ## Why use Ember
 
-In your own words, describe why someone would use Ember.
-
 ```md
-<!-- your answer here -->
+So far it seems that ember is pretty close to express and it's generator, but
+unlike express using jade, ember seems to be using handlebars as a template engine. Ember also is designed
```

Comment:
Ember is a JS framework, like Express, but it operates in a totally different domain (front end vs back end).